openedx/features/offline_mode/assets_management.py

```diff
@@ -80,33 +81,20 @@ def create_subdirectories_for_asset(file_path):
 
 def remove_old_files(xblock):
```

Comment:
Please change this function name to `clean_outdated_xblock_files`

Reply:
Changed